src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -269,7 +276,22 @@ def test_WINNF_FT_S_FAD_1(self, config_filename):
       # Very light checking of the config file.
       self.assertEqual(len(config['registrationRequests']),
                         len(config['grantRequests']))
-        
+      # check that the config file contains consistent PAL&PPA data 
+      for index, grant in enumerate(config['grantRequests']):
+        grant_frequency_range = grant['operationParam']['operationFrequencyRange']
+        for ppa in config['ppaRecords']:
+            if index in ppa['ppaClusterList']:
+              frequency_ranges_of_pals = [{'frequencyRange' : {'lowFrequency' : pal['channelAssignment']['primaryAssignment']['lowFrequency'], 'highFrequency' : pal['channelAssignment']['primaryAssignment']['highFrequency']}} \
+                for pal in config['palRecords'] if pal['palId'] in ppa['ppaRecord']['ppaInfo']['palId']]
+              self.assertTrue(frequency_ranges_of_pals)
+              low_freq = min([freq_range['lowFrequency'] for freq_range in frequency_ranges_of_pals])
+              high_freq = max([freq_range['highFrequency'] for freq_range in frequency_ranges_of_pals])
+              self.assertChannelsContainFrequencyRange( {'lowFrequency': low_freq, 'highFrequency':high_freq }, frequency_ranges_of_pals)
```

Comment:
I think instead you can check directly that all lowFreqs are the same and all highFreqs are the same.